### DISCUSSION TITLE
Choose model via Shortcut.

### DISCUSSION THREAD

**Discussion by arpagon** [briansunter/logseq-plugin-gpt3-openai#169] [Ideas]
In your last blog [post](https://briansunter.com/pages/newsletter/issue-12) , you talked about the costs of using GPT-4 vs GPT-3.5-turbo. I'd love to see this in the plugin, allowing me to choose the model via the shortcut or the command palette.

**Comment by codekiln** [comment]
Bump to this, there are significant differences between the models, and they have different use cases - https://platform.openai.com/docs/models. Preferably, the model used to generate it would be annotated in a property of the block inserted
